answer/salami/CSS/01 centerDiv/01 centerDiv.css

```
@@ -0,0 +1,6 @@
+.center{
```

```suggestion
.center {
    height: 300px;
```
みたいに `{`の前には半角スペース。ブラケットのタグの後はインデントしましょう 👀